bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/cache/ExpiringCacheMap.java

```diff
@@ -208,7 +210,7 @@ public V get(K key) {
      *
      * @return the collection of all values
      */
-    public synchronized Collection<V> values() {
+    public synchronized Collection<@Nullable V> values() {
         final Collection<V> values = new LinkedList<>();
```

Comment:
This seems like a mismatch, maybe you should not include the `@Nullable` here?

Reply:
But then we have to check if the value we add to the Collection is not `null`.

Reply:
Yes, so then 214 has to be changed as well

Reply:
I did that indeed, so a LinkedList can indeed contain `null` so its all fine.